cityofaustin/atd-moped
# Remove Data Tracker sync button

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/8536

This PR removes the current Moped-Data Tracker sync button that is replaced by the new ETL.

## Testing
**URL to test:** 

Local

**Steps to test:**
1. Start the local Moped stack from a production snapshot
2. Using the project list view to find a project with one or more signal IDs associated with it like https://localhost:3000/moped/projects/50
3. Go to the project summary page's **Signal IDs** section and notice that it shows links to the Data Tracker app signals associated with the project
4. Find a project without signals like https://localhost:3000/moped/projects/1715
5. Go to the project summary page's **Signal IDs** section and notice that it shows no signals and `-` instead
6. You can also add **Signal - Traffic** type components in the map view and then go back to the summary to see the updated signal list

---
#### Ship list
